guidelines/component-usage.md

```diff
@@ -1 +1,29 @@
+## Table of content
 
+1. [Syntax](#syntax)
+2. [Component Structure](#component-structure)
+
+## Syntax
+
+1. #### Naming Style **[⬆](#table-of-content)**
+
+   * Always use the `.js` extension.
+   * Use UpperCamelCase as a components name.
```

Comment:
component's I guess? or "for component names", to match "for prop names" below.

Reply:
Right